lib/modules/dosomething/dosomething_mbp/dosomething_mbp.install

```diff
@@ -121,6 +121,17 @@ function dosomething_mbp_update_7006(&$sandbox) {
 }
 
 /**
+ * Updates user_register production to:
+ * - include imageProcessingQueue
+ * - remove userCampaignActivityQueue
+ * from transactional_campaign_reportback production.
+ */
+function dosomething_mbp_update_7007(&$sandbox) {
+  db_drop_table('message_broker_producer_productions');
+  dosomething_mbp_install_productions();
```

Comment:
This is how it's been done all the way back to `function dosomething_mbp_update_7004(&$sandbox) {` 

Is it worth adding in a `db_table_exists()`?